cc/coord.cc

```diff
@@ -30,7 +30,7 @@ namespace {
 
 Coord TryParseKgs(absl::string_view str) {
   auto upper = absl::AsciiStrToUpper(str);
-  if (upper == "PASS") {
+  if (upper == "PASS" || upper == "TT") {
```

Comment:
I don't think this is correct: `TT` isn't a valid KGS coordinate. The GTP and KGS docs I've found all say that pass should be represented as the string `pass`. I think you can revert this particular line.

Reply:
i think "KGS" is the wrong name for these coordinates.  these are GTP coordinates and we're using this to translate our GTP commands.

So i think this should be "TryParseGtp"